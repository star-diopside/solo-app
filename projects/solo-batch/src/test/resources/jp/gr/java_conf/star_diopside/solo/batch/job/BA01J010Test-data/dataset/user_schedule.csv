user_id,schedule_id,created_at,created_user_id,updated_at,updated_user_id,version
user01,2001010100000001,1970-01-01 00:00:00.0,TEST-DATA,1970-01-01 00:00:00.0,TEST-DATA,0
user01,2001010100000002,1970-01-01 00:00:00.0,TEST-DATA,1970-01-01 00:00:00.0,TEST-DATA,1
user02,2001010100000001,1970-01-01 00:00:00.0,TEST-DATA,1970-01-01 00:00:00.0,TEST-DATA,2
user02,2001010100000002,1970-01-01 00:00:00.0,TEST-DATA,1970-01-01 00:00:00.0,TEST-DATA,3
user03,2001010100000001,1970-01-01 00:00:00.0,TEST-DATA,1970-01-01 00:00:00.0,TEST-DATA,4
user03,2001010100000002,1970-01-01 00:00:00.0,TEST-DATA,1970-01-01 00:00:00.0,TEST-DATA,5
user04,2001010100000001,1970-01-01 00:00:00.0,TEST-DATA,1970-01-01 00:00:00.0,TEST-DATA,6
user04,2001010100000002,1970-01-01 00:00:00.0,TEST-DATA,1970-01-01 00:00:00.0,TEST-DATA,7
user05,2001010100000001,1970-01-01 00:00:00.0,TEST-DATA,1970-01-01 00:00:00.0,TEST-DATA,8
user05,2001010100000002,1970-01-01 00:00:00.0,TEST-DATA,1970-01-01 00:00:00.0,TEST-DATA,9
user06,2001010100000001,1970-01-01 00:00:00.0,TEST-DATA,1970-01-01 00:00:00.0,TEST-DATA,10
user06,2001010100000002,1970-01-01 00:00:00.0,TEST-DATA,1970-01-01 00:00:00.0,TEST-DATA,11
user07,2001010100000001,1970-01-01 00:00:00.0,TEST-DATA,1970-01-01 00:00:00.0,TEST-DATA,12
user07,2001010100000002,1970-01-01 00:00:00.0,TEST-DATA,1970-01-01 00:00:00.0,TEST-DATA,13
user08,2001010100000001,1970-01-01 00:00:00.0,TEST-DATA,1970-01-01 00:00:00.0,TEST-DATA,14
user08,2001010100000002,1970-01-01 00:00:00.0,TEST-DATA,1970-01-01 00:00:00.0,TEST-DATA,15
user10,2001010100000001,1970-01-01 00:00:00.0,TEST-DATA,1970-01-01 00:00:00.0,TEST-DATA,16
