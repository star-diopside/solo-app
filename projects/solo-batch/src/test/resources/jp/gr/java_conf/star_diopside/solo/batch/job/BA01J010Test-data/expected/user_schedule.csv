user_id,schedule_id,created_at,created_user_id,updated_at,updated_user_id,version
user01,2001010100000001,1970-01-01 00:00:00.0,TEST-DATA,1970-01-01 00:00:00.0,TEST-DATA,0
user01,2001010100000002,1970-01-01 00:00:00.0,TEST-DATA,1970-01-01 00:00:00.0,TEST-DATA,1
user02,2001010100000001,1970-01-01 00:00:00.0,TEST-DATA,1970-01-01 00:00:00.0,TEST-DATA,2
user02,2001010100000002,1970-01-01 00:00:00.0,TEST-DATA,1970-01-01 00:00:00.0,TEST-DATA,3
user03,2001010100000001,1970-01-01 00:00:00.0,TEST-DATA,1970-01-01 00:00:00.0,TEST-DATA,4
user03,2001010100000002,1970-01-01 00:00:00.0,TEST-DATA,1970-01-01 00:00:00.0,TEST-DATA,5
user05,2001010100000001,1970-01-01 00:00:00.0,TEST-DATA,1970-01-01 00:00:00.0,TEST-DATA,8
user05,2001010100000002,1970-01-01 00:00:00.0,TEST-DATA,1970-01-01 00:00:00.0,TEST-DATA,9
user06,2001010100000001,1970-01-01 00:00:00.0,TEST-DATA,1970-01-01 00:00:00.0,TEST-DATA,10
user06,2001010100000002,1970-01-01 00:00:00.0,TEST-DATA,1970-01-01 00:00:00.0,TEST-DATA,11
user07,2001010100000001,1970-01-01 00:00:00.0,TEST-DATA,1970-01-01 00:00:00.0,TEST-DATA,12
user07,2001010100000002,1970-01-01 00:00:00.0,TEST-DATA,1970-01-01 00:00:00.0,TEST-DATA,13
